sal_interfaces/MTDome/MTDome_Commands.xml

```diff
@@ -80,42 +86,42 @@
   <SALCommand>
     <Subsystem>MTDome</Subsystem>
     <EFDB_Topic>MTDome_command_stopLouvers</EFDB_Topic>
-    <Description>Stop all louvers.</Description>
+    <Description>Stop all louvers. This should also be used for an emergency stop.</Description>
   </SALCommand>
   <SALCommand>
     <Subsystem>MTDome</Subsystem>
     <EFDB_Topic>MTDome_command_closeShutter</EFDB_Topic>
-    <Description>Close the shutter.</Description>
+    <Description>Close the shutter. This should also be used for an emergency stop.</Description>
   </SALCommand>
   <SALCommand>
     <Subsystem>MTDome</Subsystem>
     <EFDB_Topic>MTDome_command_openShutter</EFDB_Topic>
-    <Description>Open the shutter.</Description>
+    <Description>Open the shutter. This should also be used for an emergency stop.</Description>
   </SALCommand>
   <SALCommand>
     <Subsystem>MTDome</Subsystem>
     <EFDB_Topic>MTDome_command_stopShutter</EFDB_Topic>
-    <Description>Stop the shutter.</Description>
+    <Description>Stop the shutter. This should also be used for an emergency stop.</Description>
   </SALCommand>
   <SALCommand>
     <Subsystem>MTDome</Subsystem>
     <EFDB_Topic>MTDome_command_stopAz</EFDB_Topic>
-    <Description>Stop azimuth motion.</Description>
+    <Description>Stop azimuth motion. This should also be used for an emergency stop.</Description>
   </SALCommand>
   <SALCommand>
     <Subsystem>MTDome</Subsystem>
     <EFDB_Topic>MTDome_command_stopEl</EFDB_Topic>
-    <Description>Stop elevation (light/wind screen) motion.</Description>
+    <Description>Stop elevation (light/wind screen) motion. This should also be used for an emergency stop.</Description>
   </SALCommand>
   <SALCommand>
     <Subsystem>MTDome</Subsystem>
     <EFDB_Topic>MTDome_command_stop</EFDB_Topic>
-    <Description>Stop all motion.</Description>
+    <Description>Stop all motion. This should also be used for an emergency stop.</Description>
```

Comment:
Please rework this text to include a reference to goStationary and an explanation of why one might use one or the other of these two commands. Again, my guess is that this one stops the azimuth in a way that is abrupt and hard on the dome. If so, please consider renaming "goStationary" to "stop" and this command to "estop" or "emergencyStop". I realize it's potentially confusing to have the command differ from the low-level controller, but this is the _public_ API and it should be as clear as possible (and "goStationary" is just weird).

Reply:
Change reverted.